### PR TITLE
MEN-4987: Add U-Boot and Grub licenses to license page

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Related to, and created with: https://github.com/mendersoftware/integration/pull/1551/files

File: 302.Release-information/02.Open-source-licenses/docs.md [hunk 1]
```diff
@@ -26242,3 +26242,1035 @@ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ```
+Third party licenses used in Mender.
```
Comment (lluiscampos) [inline 302.Release-information/02.Open-source-licenses/docs.md:26245]: This header is already present at the top and should be included only once.

File: 302.Release-information/02.Open-source-licenses/docs.md [hunk 2]
```diff
@@ -26242,3 +26242,1035 @@ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
...
+
+
+--------------------------------------------------------------------------------
+
+
+git.denc.de/u-boot:
```
Comment (kacf) [inline 302.Release-information/02.Open-source-licenses/docs.md:26251]: It's "denx", not "denc".

File: 302.Release-information/02.Open-source-licenses/docs.md (hunk 1 above)
Comment (kacf) [reply 302.Release-information/02.Open-source-licenses/docs.md:26245]: Yes, and also, they need to be inside the triple backticks to be rendered inside the block.

**Review by kacf — APPROVED** [review]
Some small adjustments needed, but pre-approving.

**Pull request merged**